self-core-impl/src/main/java/com/selfxdsd/core/GithubOrganization.java

```diff
@@ -0,0 +1,54 @@
+package com.selfxdsd.core;
+
+import com.selfxdsd.api.Organization;
+import com.selfxdsd.api.Repos;
+
+import javax.json.JsonObject;
+
+/**
+ * A Github Provider Organization.
+ *
+ * @author criske
+ * @version $Id$
+ * @since 0.0.9
+ * @todo #122:30 Implement and test GithubOrganization Repos.
+ */
+public final class GithubOrganization implements Organization {
```

Comment:
@criske Make this class package-protected pls. It should not be visible outside of the core.